### PR TITLE
Add Ricoh Aficio SP C240DN driver

### PR DESCRIPTION
**Pull request by renard**
Version 2.022.02

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There’s trailing whitespace that needs to be removed.

File: Casks/ricoh-driver-aficio-sp-c240dn.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask :v1 => 'ricoh-driver-aficio-sp-c240dn' do
+  version '2.022.02'
+  sha256 'ee97bb2775f62b69785996b475ed2b599872d618c4630f543a2603ef75f73c04'
+
+  url 'http://support.ricoh.com/bb/pub_e/dr_ut_e/0001273/0001273748/V202/r69453Lf.dmg'
+  name 'Ricoh Aficio SP C240DN driver'
+  homepage 'http://support.ricoh.com/bb/html/dr_ut_e/ra/model/spc240/spc240fr.htm'
+  license :closed
```
Comment (adityadalal924) [inline Casks/ricoh-driver-aficio-sp-c240dn.rb]:
```
Casks/ricoh-driver-aficio-sp-c240dn.rb:8:18: C: Trailing whitespace detected.
  license :closed 
                 ^
```

Please fix & [squash](http://davidwalsh.name/squash-commits-git), thanks!

**Comment by renard** [comment]
Done.

BTW is there a validator something similar to check Casks befor submitting them?

**Comment by vitorgalvao** [comment]
Please [squash your commits](http://davidwalsh.name/squash-commits-git).

You can run `brew cask audit {{cask_name}}`, though that doesn’t clean trailing whitespace, if I recall correctly. You should configure your editor to clean those, though.

You can also use stuff like [`rubocop`](https://github.com/bbatsov/rubocop) to lint ruby.

**Comment by adityadalal924** [comment]
Thank you for the contribution. It needed some fixes, so they were made in d82d2d148abb8152e951adf98adf735495c9c9e4. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**